### PR TITLE
ISLE-v.1.5.5 Release - GA & Sec updates, software upgrades, release fixes

### PR DESCRIPTION
**Pull request by g7morris**
Reference https://github.com/Islandora-Collaboration-Group/ISLE/projects/5

### CODE REVIEW

**Comment by g7morris** [comment]
Good morning @marksandford & @david.keiser.clark Just an update that I'm getting through the ISLE 1.5.5 release which you can see here https://github.com/Islandora-Collaboration-Group/ISLE/pull/417/files The main takeaway is that per David's ticket here https://github.com/Islandora-Collaboration-Group/ISLE/issues/217 I took it as a branding change. `isle.localdomain` is now `isle-demo.localdomain` which is much clearer in my opinion. Also `demo` containers won't be using the opaque `ld` suffix but `demo` instead so `isle-apache-demo` instead of `isle-apache-ld`. I've updated all documentation, READMEs and am working on getting all ancillary hardcoded scripts to understand. Currently testing. I'm just commenting here to ensure that this matches your vision on the changes as well. Please let me know if I'm heading in the wrong direction or if I should continue. Thanks!

File: demo.env [hunk 1]
```diff
@@ -17,16 +17,16 @@ FEDORA_DB=fedora3
...
-FEDORA_DB_PASS=ild_feddb_2018
+FEDORA_DB_PASS=demo_feddb
 
 ### Drupal database name
-DRUPAL_DB=isle_ld
+DRUPAL_DB=isle_demo
 
 ### Drupal database user
-DRUPAL_DB_USER=isle_ld_user
+DRUPAL_DB_USER=isle_demo_user
```
Comment (noahwsmith) [inline demo.env:26]: When people pull from upstream, isn't this going to overwrite their customized values in these places? Is this juice worth the squeeze?

File: demo.env [hunk 2]
```diff
@@ -44,26 +44,52 @@ ISLE_BUILD_TOOLS_BRANCH=7.x-1.13
 ## Islandora (DRUPAL)
 
 ### Islandora's User ID (UID) - for permission issues set this to your user's `id -u`
-ISLANDORA_UID=1000
+ISLANDORA_UID=10000
+ISLANDORA_GUID=10000
 
 ### Site name
-DRUPAL_SITE_NAME=ISLE.localdomain
+DRUPAL_SITE_NAME=isle-demo.localdomain
```
Comment (noahwsmith) [inline demo.env:51]: So everyone using ISLE with the standard subdomain will need to change their /etc/hosts to have this new value

File: docker-compose.demo.yml [hunk 3]
```diff
@@ -26,7 +26,7 @@ services:
 
 
   mysql:
-    image: islandoracollabgroup/isle-mysql:1.5.4
+    image: borndigital/isle-mysql:1.5.5
```
Comment (noahwsmith) [inline docker-compose.demo.yml:29]: I assume these "borndigital" refs are temporary

**Review by noahwsmith — COMMENTED** [review]
This is a big change. Big changes make me nervous.

File: docker-compose.demo.yml (hunk 3 above)
Comment (g7morris) [reply docker-compose.demo.yml:29]: Yes just for passing the first series of tests. They'll be reverted for the release.

File: demo.env (hunk 1 above)
Comment (g7morris) [reply demo.env:26]: I guess not as I didn't realize that we've painted ourselves into a corner here. That should be just for demo purposes but instead it sounds like it is a hard coded value that can't ever be changed.

Comment (g7morris) [reply demo.env:26]: I'll revert these changes then and retest. @dwk2 I think unfortunately at this point I cannot easily change values like we thought in https://github.com/Islandora-Collaboration-Group/ISLE/issues/217 I'll retest with just the password changes but not the domain name or container changes (which are probably out of the scope of your request in any event.)

**Comment by g7morris** [comment]
Okay so after reviewing with @noahwsmith offline and in the comments, I'm going to revert changes stemming from https://github.com/Islandora-Collaboration-Group/ISLE/issues/217 On the face of it, this was a good request @dwk2  but it appears to break too much for current ISLE users and tests. I think given we're in maintenance mode, we should label this as a can't fix despite the good intentions and thinking behind it. Sorry.

File: demo.env (hunk 1 above)
Comment (dwk2) [reply demo.env]: Thank you for the careful analysis and I'm okay with these changes being reverted.

File: demo.env (hunk 2 above)
Comment (dwk2) [reply demo.env]: Thank you for the careful analysis and I'm okay with these changes being reverted.

**Comment by dwk2** [comment]
Thank you for the careful analysis and I'm okay with these changes being reverted.

**Review by dwk2 — APPROVED** [review]
Thank you for the careful analysis and I'm okay with these changes being reverted.

**Pull request merged**